src/Swashbuckle.AspNetCore.SwaggerGen/XmlComments/XmlCommentsParameterFilter.cs

```diff
@@ -52,7 +56,32 @@ private void ApplyParamTags(OpenApiParameter parameter, ParameterInfo parameterI
             if (paramNode != null)
             {
                 parameter.Description = XmlCommentsTextHelper.Humanize(paramNode.InnerXml);
+
+                var example = paramNode.GetAttribute("example", string.Empty);
+                if (!string.IsNullOrEmpty(example))
+                {
+                    parameter.Example = ConvertToOpenApiType(parameterInfo.ParameterType, parameter.Schema, example);
+                }
+            }
+        }
+
+        private static IOpenApiAny ConvertToOpenApiType(Type memberType, OpenApiSchema schema, string stringValue)
```

Comment:
Maybe you shouldn't duplicate this method and move it somewhere so it can be used here and in XmlCommentsSchemaFilter.